src/index.js

```diff
@@ -143,11 +147,13 @@ export function withField (el) {
       const { path, mapProps } = props._veeValidateConfig
       const { validations, modelValue } = toRefs(props)
       const initialValue = modelValue ? modelValue.value : undefined
+      const label = computed(() => props.label)
```

Comment:
question: Do we need this `computed` wrapper around the label prop? Props should be reactive and should not be needed now that its not an attr

Reply:
If we pass `props.label` directly it will lose reactivity. 

So we need to wrap it in a `computed` or use: `toRef`.

```js
// alternative approach
const label = toRef(props, 'label');
```